recipes/clcache/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "clcache" %}
+{% set version = "3.3.0" %}
+{% set sha256 = "d58439c548433adcda98e695be53e526ba940a4b9c44fb9a05d92cd495cdd47f" %}
```

Comment:
Looks like the sha256 is `57659520283809b96035bc1a39678588df1143435e3b3467f1fe82d564d93ad4`